# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/fastfetch/hotfix-shcpu.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- fastfetch-2.53.0/src/detection/cpu/cpu_linux.c.vanilla	2025-10-04 16:49:06.934442095 +0200
+++ fastfetch-2.53.0/src/detection/cpu/cpu_linux.c	2025-10-04 17:18:31.630000000 +0200
@@ -360,6 +360,8 @@
             (cpu->name.length == 0 && ffParsePropLine(line, "model name :", &cpu->name)) ||
             (cpu->vendor.length == 0 && ffParsePropLine(line, "vendor :", &cpu->vendor)) ||
             (cpuMHz->length == 0 && ffParsePropLine(line, "cpu MHz :", cpuMHz)) ||
+            #elif __sh__
+            (cpu->name.length == 0 && ffParsePropLine(line, "cpu type :", &cpu->name)) ||
             #else
             (cpu->name.length == 0 && ffParsePropLine(line, "model name :", &cpu->name)) ||
             (cpu->name.length == 0 && ffParsePropLine(line, "model :", &cpu->name)) ||
